README.md

```diff
@@ -1,32 +1,45 @@
 # Requiem
 
-![](https://raw.githubusercontent.com/Ladysnake/Requiem/master/requiem-logo-200x125.png)
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="requiem-logo-200x125_light.png" width="250">
+        <img src="requiem-logo-200x125.png" width="250">
+    </picture> 
+</p>
 
 [![Download](https://curse.nikky.moe/api/img/265729?logo)](https://curse.nikky.moe/api/url/265729) ![](https://jitpack.io/v/ladysnake/dissolution.svg) ![](https://img.shields.io/github/issues/ladysnake/requiem.svg)
 
+<p>
+
+[<img alt="Quilt Logo" width="400px" src="https://raw.githubusercontent.com/QuiltMC/art/master/contributed-art/doctor4t/quilt-exclusive-banner.png?raw=true" />](https://quiltmc.org/en/about/faq/#usage-questions)
+
+</p>

```

Comment:
```suggestion
```

I don't think this banner is necessary for a GitHub readme actually, curseforge and modrinth should be enough